core/server/server.js

```diff
@@ -82,6 +84,21 @@ const publicConfigSchema = Joi.object({
     prometheus: {
       enabled: Joi.boolean().required(),
```

Comment:
I think we probably should add a second prometheus flag so you can have the prometheus endpoint turned off while still collecting prometheus metrics. Something like `endpointEnabled: Joi.boolean().default(true)`. That way, we can turn off the `/metrics` endpoint after we've passed the migration period, and also new instances we set up, which don't need the Prometheus endpoint can have it turned off.

Reply:
Done. I also changed the `createTestServer` function, you can check it.